src/sentry/replays/query.py

```diff
@@ -493,3 +421,176 @@ def _activity_score():
         ],
         alias="activity",
     )
+
+
+# A mapping of marshalable fields and theirs dependencies represented as query aliases.  If a
+# column is added which depends on another column, you must add it to this mapping.
+#
+# This mapping represents the minimum number of columns required to satisfy a field.  Without this
+# mapping we would need to select all of the fields in order to ensure that dependent fields do
+# not raise exceptions when their dependencies are not included.
+#
+# If a mapping is left as `[]` the query-alias will default to the field name.
+
+FIELD_QUERY_ALIAS_MAP: Dict[str, List[str]] = {
```

Comment:
i wonder if it would possible to combine this into the `ReplayQueryConfig` itself?